book/scripts.md

```diff
@@ -56,7 +56,7 @@ When this script is run, Nushell will first run the `a` command to completion an
 
 ## Parameterizing Scripts
 
-Script files can optionally contain a special "main" command. `main` will be run after any other Nu code, and is primarily used to add parameters to scripts. You can pass arguments to scripts after the script name (`nu <script name> <script args>`).
+Script files can optionally contain a special "main" command. `main` will be run after any other Nu code, and is primarily used to add parameters to scripts, like POSIX positional parameters. You can pass arguments to scripts after the script name (`nu <script name> <script args>`).
```

Comment:
Thanks for tweaking this, I think there is nothing particularly POSIX about how we handle arguments so I would say 

`like positional parameters or flags`